docs/Guide/getting-started/creating-a-basic-app-command.mdx

```diff
@@ -76,20 +71,18 @@ export class PingCommand extends Command {
 
 :::caution
 
-Currently, progress is still being made on updating the official [Sapphire Plugin Subcommands][saph-plug-subcom]. This
-section will fully work now, and will continue to work after the plugin is updated. However, once the plugin has been
-updated creating subcommands will be possible with less code.
+Currently, progress is still being made on the official [Sapphire Subcommands Plugin][saph-plug-subcom]. The following
+example works as of now, and will continue to work after the plugin is updated. However, creating subcommands will be
+possible with less code once the plugin has been updated.
```

Comment:
the plugin has been finished. This should be adjusted to reflect that. From here link to `docs/Guide/plugins/Subcommands/getting-started`.

The `:::note` below should then also be adjusted.

Reply:
There will be another pull request with an update to the plugin subcommands guide, I'll just link to it in this pull request for now.